app/views/onestring.php

```diff
@@ -34,7 +34,7 @@
              "  <th><a href='#{$locale}'>{$locale}</a></th>\n";
 
         if (! $translation) {
-            echo "  <td><em class='error'>Warning: Missing string</em></td><td></td>\n";
+            echo "  <td><em class='error'>Warning: Empty string</em></td><td></td>\n";
```

Comment:
I’m confused why you left only “Empty”. Now on http://localhost:8082/string/?entity=dom/chrome/dom/dom.properties:GenericImageNamePNG&repo=central it’s displaying "Empty string” everywhere while the string is missing.